test/lib/ufe/testContextOps.py

```diff
@@ -726,12 +918,13 @@ def testMaterialCreationForMultipleObjects(self):
         spherePrim = usdUtils.getPrimFromSceneItem(sphereItem)
         self.assertFalse(spherePrim.HasAPI(UsdShade.MaterialBindingAPI))
 
-        # Select two of our three objects.
+        # Select all three objects.
+        ufe.GlobalSelection.get().append(capsuleItem)
         ufe.GlobalSelection.get().append(cubeItem)
         ufe.GlobalSelection.get().append(sphereItem)
 
-        # Apply the new material on the unselected object. This object should also receive the new material binding,
-        # in addition to the two selected objects.
+        # Apply the new material one of the selected objects, so we are operating in
+        # bulk mode. All the selected objects should receive the new material binding.
```

Comment:
As per design from Nat, I changed the behavior of "Assign New Material" to only work on selection (and not selection + context item).